ext/pdo_mysql/tests/pdo_mysql_stmt_fetch_serialize.phpt

```diff
@@ -79,15 +79,14 @@ MySQLPDOTest::skip();
         if (0 != $db->getAttribute(PDO::MYSQL_ATTR_DIRECT_QUERY))
             printf("[002] Unable to turn off emulated prepared statements\n");
 
-        $db->exec('DROP TABLE IF EXISTS test');
-        $db->exec(sprintf('CREATE TABLE test(id INT, myobj BLOB) ENGINE=%s',
+        $db->exec(sprintf('CREATE TABLE test_stmt_fetchserialize(id INT, myobj BLOB) ENGINE=%s',
```

Comment:
This is test is kinda pointless/outdated in its current form. What it probably should be is 3 different tests test sting the 3 different serialization mechanisms (or 4 if you cound the default one)